File: problems/o/oddecho/oddecho.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>
#include <string>

using namespace std;

int main() {
    int n;
    string word;

    getline(cin, word);
    n = stoi(word);

    for (int i=0; i<n; i++) {
        getline(cin, word);

        if (i % 2 == 0) {
            cout << word << endl;
        }
    }
}

File: problems/o/oddecho/oddecho.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    n = int(input())
    for i in range(n):
        word = input()
        if i % 2 == 0:
            print(word)


if __name__ == '__main__':
    main()